### DISCUSSION TITLE
GUI addition to DICOM Anonymization module

### DISCUSSION THREAD

**Discussion by yuvraj-wale** [Emory-HITI/Niffler#382] [Ideas] [answered]
While reviewing the codebase, I noticed comments regarding the addition of a GUI to the DICOM anonymization module. Currently, the module only offers a command-line interface for anonymizing DICOM files and addition of a GUI using tkinter could make the module more user-friendly. I'd like to confirm whether developing a GUI for the DICOM anonymization module is a valuable addition to the project. If it is, may I proceed with the development?

@pradeeban @anbhimi

**Comment by pradeeban** [comment, marked as answer]
We developed a GUI as part of the GSoC in 2021 to many modules (cold-extraction, meta-extraction, and png-extraction). It was rarely used by anyone except for the paper. So I don't find use in adding more GUI components. They just add more maintenance issues and complexities.

**Comment by yuvraj-wale** [comment]
oh ok makes sense👍🏼
